### DISCUSSION TITLE
mp3 to mp3 at same quality, split by chapters with no re-encoding?

### DISCUSSION THREAD

**Discussion by grv688** [yermak/AudioBookConverter#554] [Q&A]
I have overdrive audiobooks in mp3 format and embedded non-standard chapter markers that AudioBookConverter reads.  The books are comprised of multiple files, but are not split by chapters (each file contains multiple chapters).  I would like to essentially concatenate the mp3s then split them based on chapters read from the metadata.  I would rather not re-encode lossy mp3 to mp3 again if I can help it.  It would also be fine to create one large mp3 with embedded chapters that other players can read or have some accompanying standard chapter file like a cue sheet.  

When I test with version 6.2.10, CPU utilization is high enough for long enough that I think the program is re-encoding mp3 to mp3, even though I have chosen the same bitrate, sample rate, number of channels, and codec.  I also chose to avoid re-encoding.  

Is it actually re-encoding and is that expected behavior?  Any way to achieve what I am looking for?  I can hear the quality loss and have been looking for a solution that can avoid this but nothing elegant and simple to use has presented itself.  This application seems very close to doing what I want, but I must be missing something!  Thank you.

**Comment by grv688** [comment]
I also notice that even though I chose 64kbps in settings, the rendered files are at 128kbps.  I must be doing something wrong.

**Reply by Dreadstar69** [reply]
I had the same problem, but I made the changes to the Preset setting and them closed the program and re-started it. I verified that the setting were still what I changed them to (Constant bitrate, kb/s   = 64) and then start a conversion and the bit rate of the resulting file was 64. 

I am using version: 6.3.12.  I also check this with version 6.3.1 (latest on Steam) with the same results.
